nixos/modules/services/desktops/cpupower-gui.nix

```
@@ -0,0 +1,21 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.cpupower-gui;
+in {
+  options = {
+    services.cpupower-gui = {
+      enable = mkEnableOption "cpupower-gui";
+    };
+  };
+
+  config = mkIf cfg.enable {
+    environment.systemPackages = [ pkgs.linuxPackages.cpupower-gui ];
```

this should align with the linuxPackages that the user choses
```suggestion
    environment.systemPackages = [ config.linuxPackages.cpupower-gui ];
```

---

I get `attribute 'linuxPackages' missing` with `config` instead of `pkgs`.

---

oops, the option is:
```
boot.kernelPackages
```
so you want to reference
```
config.boot.kernelPackages
```

---

works now. Thanks!